javaagent-tooling/src/main/java/io/opentelemetry/javaagent/tooling/InstrumentationModule.java

```diff
@@ -354,8 +357,4 @@ public int getOrder() {
   protected Map<String, String> contextStore() {
     return Collections.emptyMap();
   }
-
-  protected boolean defaultEnabled() {
```

Comment:
Please leave this method -- it's useful to extend that in vendor distros.

Reply:
Thanks!

Reply:
oh my, I just saw the build failure from when I had removed it, apparently it _is_ used, bad Intellij 😂

(I updated the comment just now to remove the part about it not being used in this repo)